src/components/Assets/AddAssetComponent.jsx

```diff
@@ -23,36 +24,31 @@ const AddAssetComponent = props => (
       <div className="page-text">Fill out device specs</div>
     </div>
     <Form onSubmit={props.onCreateAsset}>
-      <label className="label-style">Model</label>
+      <div className="label-style">Model Number</div>
```

Comment:
Out of curiosity, why the change from label to div?

Reply:
At first we tried a `span` tag but the class `label-style` was not being applied. It very curiosu why it did not work. We finally decided on a `div`